services/report/__init__.py

```diff
@@ -60,6 +62,8 @@
 from services.repository import get_repo_provider_service
 from services.yaml.reader import get_paths_from_flags, read_yaml_field
 
+MIN_PARENT_COMMIT_DEPTH = 1
```

Comment:
Will this ever not be 1?

Reply:
This value itself will always be 1, but the parent depth a commit chooses is determined by this https://github.com/codecov/worker/blob/f12fe8f4701da9201ff5c64b55540b891d9cb8ea/services/report/__init__.py#L648-L649. I just made this into a constant cause I was referencing it in 2 different places